pipelined/bevy_render2/src/view/mod.rs

```diff
@@ -90,11 +90,9 @@ fn prepare_view_uniforms(
     render_device: Res<RenderDevice>,
     render_queue: Res<RenderQueue>,
     mut view_uniforms: ResMut<ViewUniforms>,
-    mut views: Query<(Entity, &ExtractedView)>,
+    views: Query<(Entity, &ExtractedView)>,
```

Comment:
How did this get past CI before? I thought clippy would deny unused mut?

Reply:
there was an `iter_mut().len()` below :)